src/core/__tests__/export-data.js

```diff
@@ -0,0 +1,621 @@
+var metadataForExport = {
```

Comment:
The data is was generated by temporarily adding code to the functions in question to write their received inputs and outputs to file, so ultimately it's from the current API and netcdf files. I wasn't entirely satisfied with this approach, but wanted us to have _some_ tests. 

I would be happy to remove them, or follow suggestions on writing them differently.